setup.py

```diff
@@ -800,10 +786,7 @@ def setupPackage():
                 'Lesser General Public License (LGPL)',
             'Operating System :: OS Independent',
             'Programming Language :: Python',
-            'Programming Language :: Python :: 2',
-            'Programming Language :: Python :: 2.7',
             'Programming Language :: Python :: 3',
-            'Programming Language :: Python :: 3.4',
             'Programming Language :: Python :: 3.5',
```

Comment:
I suggest also adding this param to `setup` to help pip know which version to install:

```python
python_requires=">=3.5",
```

Reply:
Good idea